common/country_types/00_country_types.txt

```diff
@@ -495,7 +568,76 @@ faction = {
 		needs_colony = no
 	}
 	modules = {
-		exclusive_diplomacy_module = {} # Nothing is allowed, but we need this for relationships.
+		exclusive_diplomacy_module = { # Nothing is allowed, but we need this for relationships.
+			contact_rule = on_action_only
+		}
+		basic_technology_module = {}
+	}
+}
+
+origin_dragon = { #origin_here_be_dragons dragon. Settings copied from a leviathan, for now
```

Comment:
Obviously we're going to want to clean all of these up - probably is worth asking the question if we just want to comment all of the extraneous stuff like this out entirely so as to reduce the overhead the game compiles and loads. Only argument I can think of for retaining stuff like this would be for compatibility with event mods and things, which in my opinion is not the most compelling of arguments.